app/views/details/EmploymentDatesView.scala.html

```diff
@@ -68,7 +68,7 @@ <h2 class="govuk-label--s govuk-!-margin-top-6">@{messages("employment.employmen
             id = "endAmount",
             name = "endAmount",
             legendAsHeading = true,
-            hint = Some(messages("employment.commonEmploymentDate.forExample")),
+            hint = Some(messages("employment.commonEmploymentDate.forExample", taxYear)),
```

Comment:
I guess we are missing few lines about (in the 62, I cannot reference it), but anyway we have the story of deleting this file so it does not matter.